### ISSUE TITLE
tslint cmd line return code can't be chained with ampersand

### ISSUE THREAD

**Issue by hughanderson4** [palantir/tslint#3878]
Environment: Windows 10 Developer Command Prompt.

### Bug Report

- __TSLint version__: 5.9.1
- __TypeScript version__: 2.6.2
- __Running TSLint via__: (pick one) CLI

#### TypeScript code being linted

```ts
//no space before this comment
```

with `tslint.json` configuration: 
```json
{
  "defaultSeverity": "error",
  "extends": [
    "tslint:recommended"
  ],
  "jsRules": {},
  "rules": {
    "object-literal-sort-keys": false,
    "quotemark": false,
    "no-empty": false,
    "max-classes-per-file": [ false ],
    "no-var-requires": false,
    "max-line-length": [ true, 140 ]
  },
  "rulesDirectory": [],
  "linterOptions": {
    "exclude": "src/test.ts"
  }
}
```

#### Actual behavior

I am trying to chain some command line commands together, and proceed to the next only if the tslint step succeeds.  I'm trying to read the return code from tslint and use that to continue.

```
tslint --fix --format stylish  src/**/*.ts & @echo success
ERROR: 8:3  comment-format  comment must start with a space

success
```

#### Expected behavior

Expect this to not print "success" because tslint should return 0 if an error happened.

In the above tslint command, see how it is chained to `@echo success` with the ampersand `&`
Even though an error occurred, I still see `success` echoed to command line.

When I fix the error (by adding a space to the beginning of the comment) I also see the success echoed to the console.  So effectively I am blocked from writing another command that depends on tslint success.

**Comment by giladgray** [comment]
@hughanderson4 i think you're using ampersands incorrectly in bash: http://bashitout.com/2013/05/18/Ampersands-on-the-command-line.html. 

pretty sure you want `tslint && echo success` (double-&)

**Comment by hughanderson4** [comment]
i have tried with single and double ampersand, and i get the same result either way.
i am not using bash.  i am on Windows.

**Comment by hughanderson4** [comment]
you're right.  thanks for the info.
